src/libraries/System.Private.CoreLib/src/System/Text/ASCIIUtility.cs

```diff
@@ -215,6 +215,17 @@ private static unsafe nuint GetIndexOfFirstNonAsciiByte_Default(byte* pBuffer, n
             goto Finish;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static bool ContainsNonAsciiValue(Vector128<ushort> value)
+        {
+            if (!AdvSimd.Arm64.IsSupported)
```

Comment:
Maybe add a little endian assertion check here, for self-documentation purposes?